main.star

```diff
@@ -179,7 +198,7 @@ def run(plan, args={}):
     # otherwise dummy relays spinup if chosen
     elif (
         args_with_right_defaults.mev_type
-        and args_with_right_defaults.mev_type == MOCK_MEV_TYPE
+        and args_with_right_defaults.mev_type == "mock"
```

Comment:
Would be nicer to have these constants consistently used instead of replacing them with inline strings :)

Reply:
Fixed.